src/pages/playlistDetail/PlaylistModal.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { usePlaylist } from "../../context/playlistContext/playlist-context";
+import { usePlaylistServerCall } from "../../context/playlistContext/usePlaylistServerCall";
+import "./playlistModal.css";
+
+function PlayListModal({ modal }) {
+  const { playlistState, setModal } = usePlaylist();
+  const { createPlaylist, addToPlaylist } = usePlaylistServerCall();
+  const [title, setTitle] = useState("");
+
+  const createClickHandler = () => {
+    if (title !== "") {
```

Comment:
you can directly pass title, as empty string will give falsy and wont pass the condition

Reply:
Yes, Sure